README.md

```diff
@@ -1 +1,308 @@
 # backend-capstone-template
+# ChatGPT Devs Era
+
+
+ChatGPT Devs Era is a website, customers can browse through various cooks and their menus, discovering a wide range of homemade food options. They can easily place orders for the meals they desire and have them delivered or arrange for pick-up.
+For Cooks, the platform provides a simple way to create profiles and showcase their homemade dishes. They can manage their menu, track orders, and interact with their customers.
+
+## Table of Contents
+
+- [Purpose](#purpose)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [License](#license)
+
+
+## Purpose
+
+1.	Connect customers with cooks who offer homemade food.
+2.	Support local cookers and help customers who are looking for homemade food
+
+
+
+## Features
+
+1.	Connects customers with local cooks offering homemade food.
+2.	Browse and explore a diverse range of homemade dishes.
+3.	User-friendly interface for placing and managing orders.
```

Comment:
We don't have user interface, so this line of code won't be accurate.

Reply:
yes ok